scout/server/templates/utils.html

```diff
@@ -51,17 +51,19 @@
                     {% if comment.level == 'global' %}
                       <span class="badge badge-info">GLOBAL</span>
                     {% endif %}
-                    {% if comment.created_at < case.updated_at %}
-                      <span class="badge badge-warning">OLD</span>
+                    {% if comment.created_at != case.updated_at %}
```

Comment:
Ehrm; do you really want to remove the OLD label? And should everything that is not created at update be edited?! Didn't you just invent an edited event that could be checked?

Reply:
> do you really want to remove the OLD label?

Yes, why not? If a comment is updated at a different time than when it's created I think it's fine to say that it was edited.

Keep in mind that the `comment.created_at < case.updated_at` is not working and the OLD label is never shown in current master..

> Didn't you just invent an edited event that could be checked?

Yes, but the edited event is not a comment, so it's not available here in this macro.

Reply:
> Yes, why not? If a comment is updated at a different time than when it's created I think it's fine to say that it was edited.

I'm probably missing something here, but isn't at a different time than when *the case* was updated something else? It looks like a copy-paste issue to me? Didn't you mean `comment.updated_at` or such?

> Keep in mind that the comment.created_at < case.updated_at is not working and the OLD label is never shown in current master..

Eh, what is wrong with the OLD label now? It worked like last week?

Reply:
No you are right. I've read  {% if comment.created_at != comment.updated_at %}
but it's case.updated_at, of course it didn't work. I'll fix it. Thanks!

Reply:
But why do we have the OLD badge, it is referred to the case or the comment then?

Reply:
I'm a bit confused here..

Reply:
My understanding is that the OLD label should accompany entries that were made by some user when the case looked different, and so indicate that that user may not have seen the data then that you are looking at now.

Reply:
As such, if the comment is older than the latest case update, that kind of makes sense to me?

Reply:
We did have some discussion about this regarding minor data changes that still updated the case updated_at, such as delivery report.

Reply:
Ok but then if you agree I would leave the condition:
```
{% if comment.created_at < case.updated_at %}
    <span class="badge badge-warning">OLD</span>
{% endif %}
```

and I'd add another condition immediately after:

```
{% if comment.created_at < comment.updated_at %}
    <span class="badge badge-secondary">edited</span>
{% endif %}
```

Reply:
Sounds great!